modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -2099,6 +2100,11 @@ public final class IgniteSystemProperties {
         defaults = "" + DFLT_IGNITE_USE_BINARY_ARRAYS)
     public static final String IGNITE_USE_BINARY_ARRAYS = "IGNITE_USE_BINARY_ARRAYS";
 
+    /** Enables partitions ranges in logs. */
+    @SystemProperty(value = "Enables replacing of consecutively arranged partition numbers by ranges in logs",
+        defaults = "" + DFLT_IGNITE_PRINT_PARTITION_RANGES_IN_LOGS)
+    public static final String IGNITE_PRINT_PARTITION_RANGES_IN_LOGS = "IGNITE_PRINT_PARTITION_RANGES_IN_LOGS";
+
```

Comment:
This is useless, I think.
We must log parts properly each time.

Reply:
Ok, I removed property.